zebra-consensus/src/chain.rs

```diff
@@ -204,10 +206,12 @@ where
 
     // The parameter download thread must be launched before initializing any verifiers.
     // Otherwise, the download might happen on the startup thread.
-    let groth16_download_handle = spawn_blocking(|| {
-        // The lazy static initializer does the download, if needed,
-        // and the file hash checks.
-        lazy_static::initialize(&crate::groth16::GROTH16_PARAMETERS);
+    let groth16_download_handle = spawn_blocking(move || {
+        if !debug_skip_parameter_preload {
+            // The lazy static initializer does the download, if needed,
+            // and the file hash checks.
+            lazy_static::initialize(&crate::groth16::GROTH16_PARAMETERS);
```

Comment:
Confusing Logging: Can you remove the groth16 task log from the start command, and move it here?

Otherwise, the log happens even when the task is skipped, which is confusing.

Reply:
https://github.com/ZcashFoundation/zebra/pull/3197/commits/4aadb0dea7cbad9c2bba62f3747d9655f88b23d8